File: Finalistas/finalistas.java

import java.util.Scanner;

/**
 *
 * @author Jaiderbr
 */
public class finalistas {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int test = sc.nextInt();
        while (test-- > 0) {
            int m = sc.nextInt();
            int e = sc.nextInt();
            int j = sc.nextInt();
            int d = sc.nextInt();

            int ans = 0;
            ans = Math.max(ans, e + j);
            ans = Math.max(ans, e + d);
            ans = Math.max(ans, j + d);

            if (ans >= m) {
                System.out.println("Clasificados");
            } else {
                System.out.println("Nadie clasifica");
            }
        }
    }

}

File: Finalistas/finalistas.py
def solve():
    m, e, j, d = map(int, input().split())

    ans = 0
    ans = max(ans, e + j)
    ans = max(ans, e + d)
    ans = max(ans, j + d)

    if ans >= m:
        print("Clasificados")
    else:
        print("Nadie clasifica")


def main():
    testcase = int(input())
    for _ in range(testcase):
        solve()


if __name__ == "__main__":
    main()
